src/components/VariableEditor.js

```diff
@@ -29,6 +29,7 @@ export class VariableEditor extends React.Component {
     value: PropTypes.string,
     onEdit: PropTypes.func,
     onHintInformationRender: PropTypes.func,
+    onPrettifyQuery: PropTypes.func,
```

Comment:
We shouldn't need a prettifying here, right?

Reply:
@asiandrummer, I can remove if you want, but before doing so, just making sure you got why I added it here. Let me know and I'll adjust.

Reply:
Sorry @astorije for the radio silence - after giving it more thought I'm thinking this should be okay. Go for it!